src/porepy/models/constitutive_laws.py

```diff
@@ -4477,10 +4516,16 @@ def displacement_divergence(
         div_u.set_name("div_u")
         return div_u
 
-    def biot_stabilization(self, subdomains: list[pp.Grid]) -> pp.ad.Operator:
-        """Biot stabilization term.
+    def biot_stabilization(
+        self, subdomains: list[pp.Grid], variable_name: str
```

Comment:
The exposed method mostly have only subdomains in the signature. We might want to discuss.

Reply:
agree